istio/istio.spec

```diff
@@ -521,134 +556,5 @@ sort -u -o devel.file-list devel.file-list
-
-* Fri Mar 22 2019 Brian Avery <bavery@redhat.com> - 0.10.0-2
-- Updated to 1.1.1
-
-* Fri Mar 22 2019 Brian Avery <bavery@redhat.com> - 0.10.0-1
-- Updated to 1.1
-
-* Mon Mar 4 2019 Brian Avery <bavery@redhat.com> - 0.9.0-1
-- Updated to 1.1 rc 2
-
-* Sun Feb 17 2019 Kevin Conner <kconner@redhat.com> - 0.8.0-2
-- Include authz regex header matching
-
-* Wed Feb 13 2019 Kevin Conner <kconner@redhat.com> - 0.8.0-1
-- Updated to istio 1.1.0-snapshot.6 tag
-
-* Sun Jan 20 2019 Kevin Conner <kconner@redhat.com> - 0.7.0-2
-- Updated to latest istio release-1.1 branch
-
-* Thu Jan 17 2019 Kevin Conner <kconner@redhat.com> - 0.7.0-1
-- Updated to istio release-1.1 branch and maistra 0.7.0
-
-* Wed Dec 19 2018 Kevin Conner <kconner@redhat.com> - 0.6.0
-- Updated to istio 1.0.5 and maistra 0.6.0
-
-* Thu Nov 29 2018 Kevin Conner <kconner@redhat.com> - 0.5.0
-- Update to include support for envoy route update requests
-
-* Sat Nov 24 2018 Kevin Conner <kconner@redhat.com> - 0.5.0
-- Update to include fix for RDS protocol race
-
-* Fri Nov 23 2018 Kevin Conner <kconner@redhat.com> - 0.5.0
-- Release update to maistra 0.5.0-2
-
-* Wed Nov 21 2018 Brian Avery <bavery@redhat.com> - 0.5.0
-- Release update to maistra 0.5.0-1
-
-* Mon Nov 19 2018 Brian Avery <bavery@redhat.com> - 0.5.0
-- Updated to istio 1.0.3 and maistra 0.5.0
-
-* Mon Oct 29 2018 Brian Avery <bavery@redhat.com> - 0.4.0
-- Updated to istio 1.0.3 and maistra 0.4.0
-
-* Fri Oct 12 2018 Brian Avery <bavery@redhat.com> - 0.3.0
-- Maistra 0.3.0 release, based on Istio 1.0.2
-
-* Wed Sep 12 2018 Brian Avery <bavery@redhat.com> - 0.2.0
-- Updated to istio 1.0.2 and maistra 0.2.0
-
-* Tue Sep 4 2018 Brian Avery <bavery@redhat.com> - 0.1.0
-- Stripped binaries
-
-* Tue Jul 24 2018 Brian Avery <bavery@redhat.com> - 1.0.0
-- Updated to 1.0.0-snapshot.2
-
-* Mon Jul 23 2018 Brian Avery <bavery@redhat.com> - 1.0.0
-- Updated to 1.0.0-snapshot.1
-
-* Fri Jun 29 2018 Jonh Wendell <jonh.wendell@redhat.com> - 1.0.0
-- Updated to 1.0.0-snapshot.0
-
-* Wed Apr 18 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.8.0-0
-- Vendor is not a submodule anymore
-
-* Tue Apr 17 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.8.0-0rc1
-- Update to 0.8rc1
-
-* Thu Apr 5 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.7.1-2
-- Optionally run unit tests
-
-* Sat Mar 31 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.7.1-1
-- New version
-
-* Tue Mar 13 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.6.0-4
-- Fix the docker hub and version (tag)
-
-* Mon Mar 12 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.6.0-3
-- Use /usr/local as base dir, once upstream heavily depends on that
-
-* Tue Mar 6 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.6.0-2
-- Update buildinfo file. Now it will be updated by update.sh
-
-* Tue Feb 27 2018 Jonh Wendell <jonh.wendell@redhat.com> - 0.6.0-1
-- First package
+* Mon Mar 2 2020 Brian Avery <bavery@redhat.com> - 1.1.0-1
```

Comment:
Another removal?

Reply:
This is a change pulled over from product. I think that it's worth a discussion. This is a new release moving in its own direction. Should it extend the same changelog?

Reply:
same comment as above